.github/workflows/LabelCheck.yml

```diff
@@ -0,0 +1,15 @@
+name: Check Labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
```

Comment:
I don't think we need `edited` and `synchronize` here, do we?

Reply:
Ok. This is from the template

Reply:
We definitely shouldn't need `synchronize`, because that should just occur when commits are pushed to the PR head branch.

Let's leave `edited` in for now.

Reply:
Can you also add `reopened` (in addition to `opened`)?